docs/userguide/main.md

```diff
@@ -137,6 +137,64 @@ Using `spec.general.additionalConfig` you can add settings to all nodes, using `
 
 Note that changing any of the `additionalConfig` will trigger a rolling restart of the cluster. If want to avoid that please use the [Cluster Settings API](https://opensearch.org/docs/latest/opensearch/configuration/#update-cluster-settings-using-the-api) to change them at runtime.
 
+## Configuring opensearch_dashboards.yml
```

Comment:
Looks like there was a hickup with the merging in of the newest main and some paragraphs are now duplicated. Can you please fix that?